run.py

```diff
@@ -14,4 +15,13 @@
     print(json.dumps(
         ct.country_statistic,
         indent=4,
-    ))
\ No newline at end of file
+    ))
+
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument("-j", "--json", action="store_true",
+                           help="Choose if the statistics will be saved to a json file")

```

Comment:
Please put a default output filename and let users set their own filename

Reply:
its good way 😋